client/extensions/woocommerce/state/sites/shipping-methods/selectors.js

```diff
@@ -3,15 +3,30 @@
 /**
  * External dependencies
  */
-
-import { find, get, isArray } from 'lodash';
+import { translate } from 'i18n-calypso';
+import { every, filter, find, get, isArray, some, startsWith } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import createSelector from 'lib/create-selector';
 import { getSelectedSiteId } from 'state/ui/selectors';
+import { isWcsEnabled } from 'woocommerce/state/selectors/plugins';
 import { LOADING } from 'woocommerce/state/constants';
+import {
+	isShippingMethodSchemaLoaded,
+	isShippingMethodSchemaLoading,
+} from 'woocommerce/woocommerce-services/state/shipping-method-schemas/selectors';
+
+/*
+ * By default, those methods are called "XXXX (WooCommerce Services)".
+ * We aren't pushing the "WooCommerce Services" brand anywhere in Calypso, so the method names must be changed.
+ */
+const METHOD_NAMES = {
+	wc_services_usps: translate( 'USPS', { comment: 'United States Postal Services' } ),
```

Comment:
Is there really a point in translating these? They are names of companies.

Reply:
Not really. I'll remove it in a later PR, you got here 1 minute too late :)

Reply:
Close call. Thanks!